### PR TITLE
allow multiple spaces in python_min syntax

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [x] Added a ``news`` entry
* [x] Regenerated schema JSON if schema altered (`python conda_smithy/schema.py`)

cc @beckermr

### CODE REVIEW

**Comment by beckermr** [comment]
Happy to merge once the tests pass.

File: conda_smithy/linter/hints.py [hunk 1]
```diff
@@ -247,9 +247,9 @@ def hint_noarch_python_use_python_min(
     if noarch_value == "python" and not outputs_section:
         hint = ""
         for section_name, syntax, reqs in [
-            ("host", "python {{ python_min }}", host_reqs),
-            ("run", "python >={{ python_min }}", run_reqs),
-            ("test.requires", "python {{ python_min }}", test_reqs),
+            ("host", r"python\s+{{ python_min }}", host_reqs),
+            ("run", r"python\s+>={{ python_min }}", run_reqs),
+            ("test.requires", r"python\s+{{ python_min }}", test_reqs),
```
Comment (beckermr) [inline conda_smithy/linter/hints.py:252]:
```suggestion
            ("host", r"python\s+\{\{ python_min \}\}", host_reqs),
            ("run", r"python\s+>=\{\{ python_min \}\}", run_reqs),
            ("test.requires", r"python\s+\{\{ python_min \}\}", test_reqs),
```

**Review by beckermr — COMMENTED** [review]
Maybe these need escapes?

File: conda_smithy/linter/hints.py (hunk 1 above)
Comment (jaimergp) [reply conda_smithy/linter/hints.py:252]: We might also need `\s*` because the spaces there are technically optional, IIRC

Comment (beckermr) [reply conda_smithy/linter/hints.py:252]: I don't think the spaces are optional. Also we long on that in another spot.

**Pull request merged**